### PR TITLE
Update color resource names

### PR DESCRIPTION
**Pull request by marksfoster**
Update color resource names to POR

TextOnAccentFillColorDefaultBrush > TextOnAccentFillColorSelectedTextBrush

TextOnAccentAAFillColorPrimaryBrush > TextOnAccentFillColorPrimaryBrush
TextOnAccentAAFillColorSecondaryBrush > TextOnAccentFillColorSecondaryBrush
TextOnAccentAAFillColorDisabledBrush > TextOnAccentFillColorDisabledBrush

ControlAAFillColorDefaultBrush > ControlStrongFillColorDefaultBrush
ControlAAFillColorDisabledBrush > ControlStrongFillColorDisabledBrush

**AccentFillColorDefaultBrush** > AccentFillColorSelectedTextBackgroundBrush

AccentAAFillColorDefaultBrush > **AccentFillColorDefaultBrush**
AccentAAFillColorSecondaryBrush > AccentFillColorSecondaryBrush
AccentAAFillColorTertiaryBrush > AccentFillColorTertiaryBrush
AccentAAFillColorDisabledBrush > AccentFillColorDisabledBrush

ControlStrokeColorForAAFillOnImageBrush > ControlStrokeColorForStrongFillWhenOnImageBrush

ControlAAStrokeColorDefaultBrush > ControlStrongStrokeColorDefaultBrush
ControlAAStrokeColorDisabledBrush > ControlStrongStrokeColorDisabledBrush

### CODE REVIEW

File: dev/CommonStyles/Common_themeresources_any.xaml [hunk 1]
```diff
@@ -149,27 +173,37 @@
             <SolidColorBrush x:Key="ControlOnImageFillColorSecondaryBrush" Color="{StaticResource ControlOnImageFillColorSecondary}" />
             <SolidColorBrush x:Key="ControlOnImageFillColorTertiaryBrush" Color="{StaticResource ControlOnImageFillColorTertiary}" />
             <SolidColorBrush x:Key="ControlOnImageFillColorDisabledBrush" Color="{StaticResource ControlOnImageFillColorDisabled}" />
-
-            <SolidColorBrush x:Key="AccentFillColorDefaultBrush" Color="{ThemeResource SystemAccentColor}" />
```
Comment (ranjeshj) [inline dev/CommonStyles/Common_themeresources_any.xaml:153]: >AccentFillColorDefaultBrush [](start = 36, length = 27)

@YuliaK @markfoster is the color change intentional ?

**Comment by ranjeshj** [comment]
Not sure what is going on with the radiobuttons test, it looks like the page crashed, but I'm able to download the app, run it and navigate to the page without issues.

**Comment by marksfoster** [comment]
@ranjeshj  I looked at RadioButton_themeresources.xaml and everything looks in order.

File: dev/CommonStyles/RadioButton_themeresources.xaml [hunk 2]
```diff
@@ -30,11 +30,11 @@
             <StaticResource x:Key="RadioButtonOuterEllipseCheckedStroke" ResourceKey="AccentFillColorDefaultBrush" />
             <StaticResource x:Key="RadioButtonOuterEllipseCheckedStrokePointerOver" ResourceKey="AccentFillColorSecondaryBrush" />
             <StaticResource x:Key="RadioButtonOuterEllipseCheckedStrokePressed" ResourceKey="AccentFillColorTertiaryBrush" />
-            <StaticResource x:Key="RadioButtonOuterEllipseCheckedStrokeDisabled" ResourceKey="AccentFillColorDisabledBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedStrokeDisabled" ResourceKey="AccentFillColorDisabled" />
```
Comment (karenbtlai) [inline dev/CommonStyles/RadioButton_themeresources.xaml:33]: Is there a reason why this is now referencing the colour rather than the brush?

**Review by ranjeshj — APPROVED** [review]

**Pull request merged**